server/src/tasks/eth_funder.js

```diff
@@ -0,0 +1,32 @@
+const config = require('config')
+const mongoose = require('mongoose')
+const { lockAccount, withAccount } = require('@utils/account')
+const { createNetwork } = require('@utils/web3')
+const { toWei, toChecksumAddress } = require('web3-utils')
+const EthFunding = mongoose.model('EthFunding')
+
+const ethFunder = withAccount(async (account, { accountAddress }) => {
+  try {
+    new EthFunding({
+      accountAddress: toChecksumAddress(accountAddress),
+      fundingDate: new Date()
+    }).save()
+    const bonus = config.get('bonus.eth.ropsten').toString()
+    const network = createNetwork('foreign', account)
+    const { web3 } = network
```

Comment:
worth checking here that the network is ropsten

Reply:
Yes agreed, I'll change it.